### ISSUE TITLE
Invoking with no sets spends the stack

### ISSUE THREAD

**Issue by maxvu** [alexindigo/cartesian#57]
```
$ node
> var ct = require( 'cartesian' );
undefined
> ct([]);
RangeError: Maximum call stack size exceeded
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:28:15)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
    at add (/home/max/my_projects/some-set-work/node_modules/cartesian/index.js:48:9)
> 
```

Maybe the concatenation of zero things should be zero (i.e. `[]`)?

**Comment by alexindigo** [comment]
Ha, good one. :) Thanks for the reporting.

**Comment by alexindigo** [comment]
Fixed and published `1.0.1` version. Thank you.
